docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+
+Download the [private CLI extension](https://acrcssc.z5.web.core.windows.net/acrcssc-1.1.1rc6-py3-none-any.whl) for Continuous patching 
+
+Run the following command:
+```sh
+az extension add --source <path to the CLI extension>
+```
+
+## Enable the Continuous Patching Workflow
+To enable Continuous Patching, follow the series of steps below that outline the CLI process. These guidelines detail the lifecycle of a continuous patching workflow, from its creation to subsequent updates to eventual deletion. 
+1. Login to Azure CLI with az login
+```sh
+az login
+```
+2. Login to ACR 
+```sh
+az acr login -n <myRegistry>
+```
+3. Use your preferred text editor to create a file named continuouspatching.json. Linux example below
+```sh
+vim continuouspatching.json //any file creation command and file name will do
+```
+Copy in the JSON Schema:
+```sh
+{
+    "version": "v1",
+    "tag-convention" : "<incremental|floating>",
+    "repositories": [{
+        "repository": "<Repository Name>",
+        "tags": ["<comma-separated-tags>"],   
+        "enabled": <true|false>
+    }] 
+}
+```
+The schema ingests specific repositories and tags in an array format. Each variable is defined below:
+
+- "version" allows the ACR team to track what schema version you’re on. Do not change this variable unless instructed to.
+- "tag-convention" this is an optional field. Allowed values are "incremental" or "floating" - refer to **Key Concepts** for more information.
+
+- "repositories" is an array that consists of all objects that detail repository and tag information
+    - "repository" refers to repository name
+    - "tags" is an array of tags separated by commas. The wildcard "*" can be used to signify all tags within that repository
+    - "enabled" is a Boolean value of true or false determining if the specified repo is on or off
+
+The following details an example configuration for a customer who wants to patch all tags (use the * symbol) within the repository “python”, and to patch specifically the “jammy-20240111” and “jammy-20240125” tags in the repository “ubuntu”. 
+
+JSON example:
+```json
+{
+"version": "v1",
+"tag-convention" : "incremental",
+"repositories": [{
+        "repository": "python",
+        "tags": ["*"],
+        "enabled": true
+    },
+    {
+        "repository": "ubuntu",
+        "tags": ["jammy-20240111", "jammy-20240125"],
+        "enabled": true, 
+    }]
+}
+```
+4. After creating your configuration file, it is recommended to execute a dry run to verify the intended artifacts are selected by the JSON criteria. The dry run requires a parameter called schedule, which specifies how often your continuous patching cycle will run. The schedule flag is measured in days, with a minimum value of 1 day, and a maximum value of 30 days. For example, if you want an image to be patched everyday, you would specify schedule as "1d", or 1 day. If you want a weekly patch (once a week), you would fill schedule as "7d", or 7 days. 
```

Comment:
NIT: inline code blocks for the schedule parameter.

4. After creating your configuration file, it is recommended to execute a dry run to verify the intended artifacts are selected by the JSON criteria. The dry run requires a parameter called schedule, which specifies how often your continuous patching cycle will run. The schedule flag is measured in days, with a minimum value of 1 day, and a maximum value of 30 days. For example, if you want an image to be patched everyday, you would specify schedule as `"1d"`, or 1 day. If you want a weekly patch (once a week), you would fill schedule as `"7d"`, or 7 days.